mix.exs

```diff
@@ -44,7 +44,8 @@ defmodule App.MixProject do
       {:jason, "~> 1.0"},
       {:plug_cowboy, "~> 2.0"},
       {:elixir_auth_google, git: "https://github.com/dwyl/elixir-auth-google.git", branch: "master" },
-      {:fields, git: "https://github.com/dwyl/fields.git", branch: "update-ecto"}
+      {:fields, path: "/home/simon/Documents/dwyl/fields"}
```

Comment:
this won't work very well on CI ... 😜 
(_but I get it that you're testing ..._) 👍

Reply:
arf yes, well spotted, I'll add back the github link.
The Fields PR is now merged

Reply:
@SimonLab thanks for merging the PR. 👍 
I will add "_enough_" docs to **`fields`** to publish it to Hex.pm 📦 
so we no longer have to use the GitHub link. 🙄